zipline/finance/performance/position_tracker.py

```diff
@@ -204,13 +216,7 @@ def auto_close_position_events(self, next_trading_day):
             past_asset_end_dates.add(date)
 
             for sid in sids:
-                # Yield a CLOSE_POSITION event
-                event = Event({
-                    'dt': date,
-                    'type': DATASOURCE_TYPE.CLOSE_POSITION,
-                    'sid': sid,
-                })
-                yield event
+                yield generate_close_event(sid, date)
```

Comment:
The verb `generate` here makes me think that `generate_close_event` is a [generator function](https://docs.python.org/2/tutorial/classes.html#generators).